css/style.css

```diff
@@ -0,0 +1,28 @@
+.date {
+  position: relative;
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #aaacbc;
+}
+
+.hide {
+  display: none;
+}
+input {
+  height: 30px;
+}
+
+#inputTask input,
+#inputTask button,
+#inputTask .trashInput {
```

Comment:
Although this project might be an exception, let's try to avoid using IDs as CSS selectors if possible.

Reply:
done